iModelCore/GeomLibs/PublicAPI/GeomSerialization/GeomLibsFlatBufferApi.h

```diff
@@ -22,25 +22,55 @@ BEGIN_BENTLEY_GEOMETRY_NAMESPACE
 class BentleyGeometryFlatBuffer
 {
 private:
-BentleyGeometryFlatBuffer (){} // no intances.
+BentleyGeometryFlatBuffer (){} // no instances.
+// template <class PtrName, class ReaderMethodName>
+// PtrName BytesToXXXSafe(Byte const *buffer, size_t const bufferSize, bool applyValidation);

```

Comment:
remove dead comments

Reply:
Thanks, will remove the dead code. The PR was not 100% ready to push. That's why it's a draft.
Also the white space removal is because I have this feature enabled automatically on save.